### PR TITLE
BC1-63-complete-html-css-for-registerpage

### PR DESCRIPTION
**Pull request by sinaalimi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,27 +1,324 @@
 <!DOCTYPE html>
```
Comment (abdolmajed3) [inline index.html:1]: why the main index file is changed?
i think something wrong happened.

**Review by abdolmajed3 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (sinaalimi) [reply index.html:1]: There has been no change and this part is unrelated to my request.

**Comment by abdolmajed3** [comment]
there are some conflict. please resolve it.

File: register/index.html [hunk 2]
```diff
@@ -1,40 +1,71 @@
...
+    <link rel="apple-touch-icon-precomposed"sizes="72x72"href="img/apple-touch-icon-72x72.png"/>
+    <link rel="stylesheet" href="scss/lib/tiny-slider.css" />
+    <link rel="stylesheet" href="css/style.css" media="all" />
+  </head>
+  <body>
+    <section class="register-form">
+      <img src="img/bg-registerpage.jpg" alt="loptop" class="pic-register">
+      <div class="section-login">
+        <form class="section-label-input">
+          <spam>Sign up</spam>
```
Comment (abdolmajed3) [inline register/index.html:38]: I think this is span tag not spam. right?

File: register/scss/layout/register-page.scss [hunk 3]
```diff
@@ -1,3 +1,162 @@
...
+      justify-content         : center;
+      display                 : flex;
+
+      @include tablet{
+        width                 : 100%;
+      }
+
+      spam{
+          padding             : 1em;
+          font-size           : 3em;
```
Comment (abdolmajed3) [inline register/scss/layout/register-page.scss:40]: you can use shared codes for spacing.

File: register/scss/layout/register-page.scss [hunk 4]
```diff
@@ -1,3 +1,162 @@
...
+
+            }
+          }
+          
+
+          label{  
+            font-weight         : bold;
+            font-size           : 20px;
+            display             : flex;
+            margin              : 1em 0em;
```
Comment (abdolmajed3) [inline register/scss/layout/register-page.scss:74]: we have standard font sizes and also spacing.

**Review by abdolmajed3 — CHANGES_REQUESTED** [review]

**Review by abdolmajed3 — APPROVED** [review]
Nice job bro

**Pull request merged**